### DISCUSSION TITLE
Does this support deserialization for APIs?

### DISCUSSION THREAD

**Discussion by dessalines** [bloodyowl/boxed#27] [Ideas]
I'd like to use these Option types in an API, IE currently an interface might be:

```ts
interface Person {
  name: string;
  display_name?: string // I'd like this to be display_name: Option<string>;
  ...
}
```

**Comment by bloodyowl** [comment]
Hi!

We don't yet provide this kind of feature, but that's something we'll be exploring in the future 🙂
